.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+**Issue**: Resolves #
+
+**[CVSS v3 Mapping](https://github.com/bugcrowd/vulnerability-rating-taxonomy/blob/master/mappings/cvss_v3.json)**:
+
+_If needed_
+
+**[Deprecated Node Mapping](https://github.com/bugcrowd/vulnerability-rating-taxonomy/blob/master/deprecated-node-mapping.json)**:
```

Comment:
`_If needed_` as well

Reply:
the if needed is for the deprecated node mapping. I want people to call out the CVSS mappings always so it's easy to review.

Reply:
if that's the case put it under the headings, that's a little confusing